ui/styles/theme.go

```diff
@@ -18,23 +24,43 @@ type Theme struct {
 	SubleMainText      lipgloss.AdaptiveColor
 }
 
-var subtleIndigo = lipgloss.AdaptiveColor{Light: "#5A57B5", Dark: "#242347"}
+var DefaultTheme = func() Theme {
+	_shimHex := func(hex config.HexColor) lipgloss.AdaptiveColor {
+		return lipgloss.AdaptiveColor{Light: string(hex), Dark: string(hex)}
```

Comment:
I don't fully understand how everything fits together with the bubbletea stuff, so this is another instance of me being careful to not change any interfaces. This is also why I wrote that custom unmarshalling function in the config parser - to not break stuff.

If we're defaulting to a combination of ANSI colors that works for both light + dark mode and make the whole theme part optional, it could be written in a lot less clumsy fashion, for sure. I guess when I'm free I'll play around with ANSI colors a bit and drop the default theme as per your suggestion, that'd adress most of the points you brought up. Is there anything you'd do differently with regards to the representation of themes in yaml, e.g. keep the original "flat" structure? I'm pretty susceptible to making things too complicated.

Thanks for the review :)